README.md

```diff
@@ -16,18 +16,19 @@ This is an early version of the plugin, my primary goal was to overcome the bann
 
 **Currently working on Android only, but iOS support will be available so soon.**
```

Comment:
You can update this line now :)

Reply:
@c4talyst thanks for looking through this. Most of it was written sleep deprived.

Reply:
I'm no expert here, but will happily review the readme! :) I'll try give the branch a test too

Reply:
@c4talyst please do run the branch! I haven't fully flushed out differ banner sizes.